### PR TITLE
docs: upgrade to 1.8.0

### PR DESCRIPTION
**Pull request by milasuperstar**
Upgrade docs to 1.8.0 plus changelog :)

Signed-off-by: Mila Wu <wumila1996@gmail.com>

### CODE REVIEW

**Review by boojack — APPROVED** [review]

**Review by LiuJi-Jim — APPROVED** [review]
LGTM

File: content/changelog/bytebase-1-8-0.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## 🎠 Community
+
+- Thanks to @[Zheaoli](https://github.com/Zheaoli) chore: fix dataDir format error when the `--data` arguments is `/` [#3176](https://github.com/bytebase/bytebase/pull/3176)
+
+## 📰 Fresh Off the Press
+
+- [The SQL Review Tool for Developers](https://www.bytebase.com/blog/sql-review-tool-for-devs)
+- [Introducing DB Cost](https://www.bytebase.com/blog/introducing-dbcost), the simple cloud database pricing sheet for AWS RDS and Google Cloud SQL instance.
+- [What is a Database Schema?](https://www.bytebase.com/blog/what-is-database-schema)
+- An oldie but goodie: [Database Review 2021 and 2022 Predictions](https://www.bytebase.com/blog/database-review-2021)
```
Comment (tianzhou) [inline content/changelog/bytebase-1-8-0.md:44]:
```suggestion
- [The SQL Review Tool for Developers](/blog/sql-review-tool-for-devs)
- [Introducing DB Cost](/blog/introducing-dbcost), the simple cloud database pricing sheet for AWS RDS and Google Cloud SQL instance.
- [What is a Database Schema?](/blog/what-is-database-schema)
- An oldie but goodie: [Database Review 2021 and 2022 Predictions](/blog/database-review-2021)
```

**Review by tianzhou — APPROVED** [review]
LGTM

**Review by ecmadao — APPROVED** [review]

File: content/changelog/bytebase-1-8-0.md (hunk 1 above)
Comment (milasuperstar) [reply content/changelog/bytebase-1-8-0.md:44]: right i always forget about these yikes

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**